src/lsm/compaction.zig

```diff
@@ -133,7 +154,8 @@ pub fn CompactionType(
                 pending: usize,
             },
             next_tick,
-            done,
+            done_writing_tables,
+            done_applied_manifest,
```

Comment:
```suggestion
            wrote_tables,
            applied_to_manifest,
```

To keep the tense consistent.

Would be great to also fix up the other enums here. For example `iter_init_a` should be `iterator` instead of `iter` (no abbreviations per TigerStyle, unless we're using `it` in a local context).

Even better, how can we use MSB order, and relate all the enums to each other? For example, `tables_writing` and `tables_written` (or `tables_writing_done`) instead of `writing` and `done_writing_tables`?